resources/views/admin/emails/example.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends('admin.emails.main')
+
+@section('css')
```

Comment:
I'm not sure that email templates can use separate CSS.
They only use `<style></style>` tags and inline styles.
Please investigate HTML email rules.